dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -454,7 +454,7 @@ def color_for_status(status)
 end
 
 Then /^element "([^"]*)" does not exist/ do |selector|
-  @browser.execute_script("return $(#{selector.dump}).length").should eq 0
+  expect(@browser.execute_script("return $(#{selector.dump}).length")).to eq 0
```

Comment:
Note: Switching to rspec's new expect syntax here (the `should` syntax is deprecated, we've been getting warnings about it).